app/models/events/review.rb

```diff
@@ -37,6 +39,7 @@ class Review < ApplicationRecord
                        inverse_of: :owned_reviews
     belongs_to :pull_request, class_name: 'Events::PullRequest',
                               inverse_of: :reviews
+    belongs_to :review_request
```

Comment:
This is optional right? What if I review a PR without being requested?

Reply:
AFAIK once you add a review, it adds you in the requested reviews list, so it triggers two events. The only problem that I see would be if the review creation goes before the review request creation. Maybe we should prioritize the jobs?

Reply:
I don't think prioritizing the jobs is guaranteed to fix the problem. Would there be a problem if we leave the association as optional? Do we use it for something?

Reply:
I found it necessary when it comes to find when the review was asked. For example: `review.created_at - review.review_requested.created_at` to get how much did the user last to make the review.

Reply:
If there's no review request, then there's no review turnaround to measure, so we can ignore those reviews while calculating the metric.